src/Categories/Bicategory/LocalCoequalizers.agda

```diff
@@ -0,0 +1,46 @@
+{-# OPTIONS --without-K --safe #-}
+
+open import Categories.Bicategory
+
+module Categories.Bicategory.LocalCoequalizers {o ℓ e t} (𝒞 : Bicategory o ℓ e t)  where
+
+open import Categories.Diagram.Coequalizer
+open import Level
+open import Categories.Functor.Properties
+import Categories.Bicategory.Extras as Bicat
+open Bicat 𝒞
+open import Categories.Functor
+
+postCompFunc : {A B C : Obj} → B ⇒₁ C → Functor (hom A B) (hom A C)
+postCompFunc = _⊚-
+
+preCompFunc : {A B C : Obj} → A ⇒₁ B → Functor (hom B C) (hom A C)
+preCompFunc = -⊚_
+
+record LocalCoequalizers : Set (o ⊔ ℓ ⊔ e ⊔ t) where
+  field
+    localCoequalizers : (A B : Obj) → Coequalizers (hom A B)
+    precompPreservesCoequalizer : {A B C : Obj} → (f : A ⇒₁ B)
+      → PreservesCoequalizers (preCompFunc {A} {B} {C} f)
+    postcompPreservesCoequalizer : {A B C : Obj} → (f : B ⇒₁ C)
+      → PreservesCoequalizers (postCompFunc {A} {B} {C} f)
+      
+  precompCoequalizer : {A B C : Obj} → {X Y : B ⇒₁ C} {α β : X ⇒₂ Y}
```

Comment:
Feels like maybe this deserves a short, infix name, like `_⊚◁_` or something like that. [Emphasis on 'like that']